provisioning/plugins/corim-psa-decoder/test-harness/build-test-vectors.sh

```diff
@@ -7,15 +7,15 @@ set -o pipefail
 
 # function generate_go_test_vector constructs CBOR test vector using
 # supplied comid and corim json template and saves them in a file
-# $1 file name for comid json template
-# $2 file name for corim json template
+# $1 file name for comid json template, example one of COMID_TEMPLATES
+# $2 file name for corim json template, example CORIM_CCA_TEMPLATE
 # $3 a qualifier for each cbor test vector name
 # $4 name of the file where the generated CBOR test vectors are aggregated
 generate_go_test_vector () {
 	echo "generating test vector using $1 $2"
 	cocli comid create -t ${1}.json
 	cocli corim create -m ${1}.cbor -t $2 -o corim${1}.cbor
-	echo "// automatically generated from $t.json" >> ${4}
+	echo "// automatically generated from $1.json" >> ${4}
```

Comment:
what I meant is

```suggestion
        echo "// automatically generated from:" >> ${4}
        echo "// - $1, and" >> ${4}
        echo "// - $2" >> ${4}
```

Reply:
I think the argument comment makes it clear what each argument means so it is quite evident

Reply:
OK, Have tidied it a bit further and checked the file looks neat and tidy now (I mean the test_vectors.go)